__tests__/app.test.js

```diff
@@ -256,12 +256,117 @@ describe("POST /api/articles/:article_id/comments", () => {
   });
 
   test("400 error for empty body", () => {
-    const Newcomment = {
-      username: "butter_bridge"
+    const NewComment = {
+      username: "butter_bridge",
     };
     return request(app)
       .post("/api/articles/1/comments")
-      .send(Newcomment)
+      .send(NewComment)
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("400 Bad Request");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id will indicate how much the votes property in the database should be updated by", () => {
+  test("should respond with a 201 status and an updated article", () => {
+    const newArticle = {
+      inc_votes: 1,
+    };
+    return request(app)
+      .patch("/api/article/1")
+      .send(newArticle)
+      .expect(201)
```

Comment:
a status code of 200 would be more appropriate here as the article has already been created - this is updating it